lib/utils/isFirstNestedStatement.js

```diff
@@ -0,0 +1,14 @@
+/* @flow */
+"use strict"
+
+module.exports = function (statement/*: postcss$rule | postcss$atRule*/)/*: boolean*/ {
+  if (statement.type !== "rule" && statement.type !== "atrule") {
```

Comment:
Not only `at-rule-empty-line-before`, `rule-nested-empty-line-before` have `first-nested` option. `comment-empty-line-before`, `custom-property-empty-line-before`, `declaration-empty-line-before` have it also. This helper can be used for more rules.